demo/number-field.html

```diff
@@ -0,0 +1,42 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1, minimum-scale=1" />
```

Comment:
Issues like this should be fixed in skeleton and then applied w/o CR to all components we have: https://github.com/vaadin/vaadin-element-skeleton/blob/master/demo/advanced.html